.circleci/config.yml

```diff
@@ -123,3 +135,11 @@ workflows:
       #- coveralls:
           #requires:
             #- build-and-test
+
+  check-version-updated:
+    jobs:
+      - check-version-updated:
+          filters:
+            branches:
+              only:
+                - /^(release|hotfix)/.*/
```

Comment:
So all branches with /release/ or /hotfix/ in its name will have to update their version?

Reply:
yes, that's the idea

Reply:
this is to ensure that we are going to have the version update in files when release without need to push to master (we do not want that) or a new PR